.dockerignore

```diff
@@ -2,14 +2,16 @@
 /bm
 dist
 
+# Immudb data
+/data*
+
 # Vendor
 vendor
 
 # Unuseful stuff
 *.iml
 *.swp
 .idea/
-.git/
```

Comment:
just wondering why `.git` can be removed from the list

Reply:
We un-ignore the .git folder so that when the COPY is performed into the initial docker image, the `.git` folder is copied too.

Without this, `make` is not able to extract current git hash and thus the version info is incomplete.

Stefano just added displaying this hash in webconsole next to the version info (so that we know exactly what commit was compiled) to ease troubleshooting.

Alternative would be to pass the commit through docker build args but that would be much larger change.

Reply:
probably after this PR:

it'd be better to build docker images by fetching already notarized immudb binaries

and we could expose the build info from immudb so the webconsole or any other app could consume it

Reply:
build info is (partially, only the version and git hash appended to it) available in the `HealthCheck` rpc call, webconsole will be calling this API to get the data.